modules/olmv1-catalog-plain.adoc

```diff
@@ -0,0 +1,263 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-managing-catalogs.adoc
+
+:_content-type: CONCEPT
+
+[id="olmv1-building-plain-bundle-image-source_{context}"]
+= Building a plain bundle image from an image source
+
+The Operator Controller currently only supports installing plain bundles created from a _plain bundle image_.
```

Comment:
In the upstream doc, I did not follow ISG guidance on the usage of "only". 

Just to clarify, it should read something like: Currently, the Operator Controller supports installing plain bundles created only from a _plain bundle image_. 

Or maybe reworked to avoid only, but to get the same idea across. There are plans to support plain bundles for different images and I think Git repos on the road map.